parser/fileparser.go

```diff
@@ -35,89 +36,31 @@ func readDir(cpath string, logger *log.Logger) []string {
 		// if file.IsDir() && file.Mode() != os.ModeSymlink {
 		// 	toReturn = append(toReturn, readDir(path.Join(cpath, file.Name()), logger)...)
 		// }
-		if !file.IsDir() && strings.HasSuffix(file.Name(), "gz") ||
-			strings.HasSuffix(file.Name(), "log") {
+		if !file.IsDir() && strings.HasSuffix(file.Name(), ".gz") ||
+			strings.HasSuffix(file.Name(), ".log") {
 			toReturn = append(toReturn, path.Join(cpath, file.Name()))
 		}
 	}
 	return toReturn
 }
 
-// readDirRolling recursively reads the directory looking for log and .gz files
-// that match the timestamps of the associated chunk
-func readDirRolling(currentChunk int, totalChunks int, cpath string, logger *log.Logger) []string {
+// readFiles reads the files and directories looking for log and gz files
+func readFiles(paths []string, logger *log.Logger) []string {
```

Comment:
By allowing the user to specify individual files on the command line to import we can get rid of this problematic code that deals with parsing times out of filenames. It also lets us use the same code for both rolling and non-rolling imports.